src/Compilers/VisualBasic/Portable/Emit/NoPia/EmbeddedTypesManager.vb

```diff
@@ -194,6 +194,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Emit.NoPia
                     TypeKind.Enum,
                     TypeKind.Delegate
 
+                    If type.IsTupleType Then
+                        type = type.TupleUnderlyingType
```

Comment:
I'd rather not do any refactorings of the changes I'm porting, or write new tests for those changes. Please file an issue if you'd like that done.